### DISCUSSION TITLE
Jobrunr Evaluation

### DISCUSSION THREAD

**Discussion by Somraj-Seal** [jobrunr/jobrunr#314] [Q&A]
We are currently evaluating Jobrunr

i have my job setup like this (Its a recurring job)

@job(name = "My Job", retries = 1)

After my job throws exception, though the retry is set to 1, it keeps on retrying. Am i missing any configurations ?

@rdehuyss Any information on this would be very helpful.

Thanks

**Comment by rdehuyss** [comment]
Hi @Somraj-Seal ,

Although this is tested already by automated tests, I did test it again manually and everything works as expected.

Please setup a Github repo that reproduces the problem.

See also https://github.com/jobrunr/jobrunr/blob/master/core/src/test/java/org/jobrunr/jobs/filters/RetryFilterTest.java#L72-L87

Please also do not open the same question twice.

Kind regards,
Ronald

**Reply by Somraj-Seal** [reply]
Hi @rdehuyss ,

Thanks very much for the prompt reply. 

I think i get it now. "retries" is working as expected. I was not able to differentiate between a retry and a recurring run of the job.

Thanks once again !!!
